src/components/formStore/table/useFormStoreTable.tsx

```diff
@@ -391,11 +391,27 @@ export default function useFormStoreTable({
     submissionIdValidationMessage,
   ])
 
+  const defaultHiddenColumns = [
+    'SUBMISSION_ID',
+    'EXTERNAL_ID',
+    'TASK',
+    'TASK_ACTION',
+    'TASK_GROUP',
+    'TASK_GROUP_INSTANCE',
+  ]
+
   const [initialState] = React.useState<Partial<TableState<FormStoreRecord>>>(
     () => {
       const text = localStorage.getItem(localStorageKey(form.id))
       if (text) {
-        return JSON.parse(text)
+        const intialState = JSON.parse(text)
+
+        if (intialState && !intialState.defaultHiddenColumnsUpToDate) {
+          intialState.hiddenColumns = defaultHiddenColumns
+          intialState.defaultHiddenColumnsUpToDate = true
```

Comment:
Might be worth making this a string with a version e.g. `V1`. That way we can change to `V2` later if we want to add more default hidden columns. And `V3` later again, etc.

Reply:
yeah good idea, will update 👍